components-core/src/main/java/com/adyen/checkout/components/status/api/StatusService.kt

```diff
@@ -27,7 +26,7 @@ internal class StatusService {
         val body = StatusRequest.SERIALIZER.serialize(statusRequest).toString()
 
         val httpClient = HttpClientFactory.getHttpClient(url)
-        val bytes = httpClient.post("", body, CONTENT_TYPE_JSON_HEADER)
+        val bytes = httpClient.post("", body)
```

Comment:
We should refactor this next, the `StatusApi` is directly building the url which is why we post with an empty string here.

Reply:
Sure, will take a look at this next